nondet/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "nondet"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+solana-program = "1.14.10"
```

Comment:
basic `nondet` should not have any dependence on Solana program.

We specifically do not want to depend on specific versions of `solana-program` since these get stale rather quickly

Reply:
I agree. However, we are trying to implement `nondet` for Solana's `Pubkey` and `AccountInfo`:

```
// need to implenet Nondet for Pubkey and AccountInfo here instead of solana/src/lib.rs
// because we can only implement a trait for an arbitrary type in the crate where the trait is defined
nondet_impl! {Pubkey, CVT_nondet_pubkey(), "Nondet for Pubkey" }
nondet_impl! {AccountInfo<'static>, CVT_nondet_account_info(), "Nondet for AccountInfo" }
```

I can try separating it out into "basic" and "non-basic" `nondet`? lets discuss this more ?

Reply:
See how `Arbitrary` implemented. We can't have all implementation of `Nondet` be in one crate. This will not be sustainable.